.github/dependabot.yml

```
@@ -0,0 +1,7 @@
+version: 2
+updates:
+- package-ecosystem: nuget
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
```

Let's point dependabot to the `/samples` directory so that it checks all of the NuGet packages in `CommunityToolkit.Maui.Sample.sln`, and let's remove the `open-pull-requests-limit`

```suggestion
version: 2
updates:
- package-ecosystem: nuget
  directory: "/samples"
  schedule:
    interval: daily
```